feature-libs/cart/import-export/components/export-entries/export-entries.service.ts

```diff
@@ -100,6 +108,19 @@ export class ExportEntriesService {
     );
   }
 
+  protected displayExportMessage() {
+    this.translationService
+      .translate('exportEntries.exportMessage')
+      .pipe(first())
+      .subscribe((message) =>
+        this.globalMessageService.add(
+          message,
+          GlobalMessageType.MSG_TYPE_INFO,
+          this.exportConfig.messageTimeout
```

Comment:
why do you like to override the default timeout for INFO in this case?

Reply:
Yes, by default there is 3000 milliseconds and I think it's not enough time to read the content as the text is quite long (especially first time when user might not expect any message). That's why I would like to have it configurable.